server/build/plugins/pages-manifest-plugin.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import { RawSource } from 'webpack-sources'
+import { MATCH_ROUTE_NAME } from '../../utils'
+import {PAGES_MANIFEST} from '../../../lib/constants'
+
+export default class PagesManifestPlugin {
```

Comment:
Shall we add a comment on what this plugin is about

Reply:
Done